src/transformers/models/roberta_prelayernorm/configuration_roberta_prelayernorm.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# Copyright 2022 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" RoBERTa-PreLayerNorm configuration"""
+from collections import OrderedDict
+from typing import Mapping
+
+from ...onnx import OnnxConfig
+from ...utils import logging
+from ..bert.configuration_bert import BertConfig
+
+
+logger = logging.get_logger(__name__)
+
+ROBERTA_PRELAYERNORM_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "andreasmadsen/efficient_mlm_m0.40": (
+        "https://huggingface.co/andreasmadsen/efficient_mlm_m0.40/resolve/main/config.json"
+    ),
+}
+
+
+class RobertaPreLayerNormConfig(BertConfig):
```

Comment:
Same comment about inheritance. I know the model is the same but feel like we want to have independence right @sgugger ?

Reply:
I updated this. It did take a bit more work because of the checkpoint paths.